npc/quests/quests_13_1.txt

```diff
@@ -8888,10 +8888,7 @@ mid_campin,90,121,5	script	Hibba Agip	459,{
 		mes "And please take this small reward.";
 		mes "Thank you for your help so far.";
 		next;
-		if (checkre(3))
-			getexp 300000,0;
-		else
-			getexp 2500000,0;
+		getexp (checkre(3))?300000:250000,0;
```

Comment:
Is that change really needed instead of simply removing one zero in pre-renewal?
This way you cannot see what was fixed clearly.

Reply:
No that's because it gives no job exp, so you can optimize it in that way, and also to maintain the slyle of the rest of the file.